src/easy/handler.rs

```diff
@@ -807,6 +807,25 @@ impl<H> Easy2<H> {
         }
     }
 
+    /// Provides the ABSTRACT UNIX SOCKET which this handle will work with.
+    ///
+    /// The `addr` param can be any bytes.
+    ///
+    /// This function is an alternative to [`Easy2::unix_socket`] and [`Easy2::unix_socket_path`] that supports
+    /// ABSTRACT_UNIX_SOCKET(`man 7 unix` on Linux) address.
+    ///
+    /// By default this option is not set and corresponds to
+    /// [`CURLOPT_ABSTRACT_UNIX_SOCKET`](https://curl.haxx.se/libcurl/c/CURLOPT_ABSTRACT_UNIX_SOCKET.html).
+    pub fn abstract_unix_socket(&mut self, addr: &[u8]) -> Result<(), Error> {
```

Comment:
Make sure you expose the same option in `Easy` in `handle.rs` as well!

Reply:
done